src/style.css

```
@@ -1 +1,70 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+html {
+  font-family: "Roboto", Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.nav {
+  display: flex;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+  --color-active-hover: #00acdc;
+}
+
+.header__logo-link {
+  margin-top: 10px;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__item {
+  display: inline-block;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  text-transform: uppercase;
+  padding: 0;
+}
+
+.nav__link {
+  display: flex;
+  text-decoration: none;
+  align-items: center;
+  color: black;
+  font-size: 12px;
+  height: 60px;
```

do not hardcode heigth use verical paddings instead 
```suggestion
  padding: 23px 0; 
```